content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -307,13 +317,13 @@ eas login
 Let’s create our app with the following:
 
 ```bash
-npx create-expo-app -t expo-template-blank-typescript solana-expo
+npx create-expo-app --template blank-typescript solana-expo
 cd solana-expo
+npx expo install expo-dev-client # A library that allows creating a development build and includes useful development tools. It is optional but recommended.
```

Comment:
This sentence is confusing. "A library that allows creating a development build" - can you re-word?